src/page-templates/content-page/content-page-template.tsx

```diff
@@ -647,7 +647,13 @@ const ContentPageTemplate: NextPage<ContentPageProps> = ({
                                     {topic.name}
                                 </TypographyScale>
                             </a>
-                            <FollowLink topic={topic} iconsOnly />
+                            {
+                                // Render FollowLink only if the topic is followable
+                                // This is for item without primary tag (e.g., Podcast, Video, Event)
+                                topic.type === 'ContentType' ? null : (
```

Comment:
I think we could go further with this. I believe the only content types we want them to be able to follow are L1/L2 products, Technologies, and Programming Languages, so it might be best to just explicitly limit it to those.